src/utils/function.js

```diff
@@ -24,7 +24,7 @@
  * @param {number} minInterval the minimum time interval in millisecond
  * @returns {function()}
  */
-export function rateLimit(win, callback, minInterval) {
```

Comment:
Then why's `debounce` in there? And they're so similar they should be in the same file.